src/ray/rpc/grpc_client.h

```diff
@@ -186,15 +197,20 @@ class GrpcClient {
   }
 
  private:
+  static bool TestingRpcFailureAvoidIntraNodeFailures(std::string_view server_address,
+                                                      std::string_view local_address) {
+    return ::RayConfig::instance().testing_rpc_failure_avoid_intra_node_failures() &&
+           (server_address == "127.0.0.1" || server_address == local_address);
+  }
+
   ClientCallManager &client_call_manager_;
   /// The channel of the stub.
   std::shared_ptr<grpc::Channel> channel_;
   /// The gRPC-generated stub.
   std::unique_ptr<typename GrpcService::Stub> stub_;
   /// Whether CallMethod is invoked.
   std::atomic<bool> call_method_invoked_ = false;
-  /// Whether to use TLS.
-  bool use_tls_;
+  bool testing_rpc_failure_avoid_intra_node_failures_ = false;
```

Comment:
```suggestion
  bool skip/avoid_testing_rpc_failure_ = false;
```

The original name sounds like a config and doesn't convey the message that we have already checked that local and remote are on the same node.

Reply:
ya makes sense changed to skip_testing_intra_node_rpc_failure_, at least keeps the intra node part in there